entries/p/pcpartpicker.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "PCPartPicker": {
+    "domain": "pcpartpicker.com",
+    "contact": {
+      "twitter": "PCPartPicker",
+      "facebook": "pcpartpicker"
+    },
+    "keywords": [
+      "utilities"

```

Comment:
The utilities category is meant for ISPs,phone service providers, gas providers etc.